server/game/cards/15-W2D/ReggieMallard.js

```diff
@@ -0,0 +1,28 @@
+const DudeCard = require('../../dudecard.js');
+
+class ReggieMallard extends DudeCard {
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            match: this,
+            condition: () => this.game.shootout && this.isParticipating(),
+            effect: ability.effects.dynamicBullets(() => this.getHucksterDudesInPosse())
+        });
+    }
+
+    getHucksterDudesInPosse() {
+        if(!this.game.shootout) {
+            return 0;
+        }
+        const Posse = this.game.shootout.getPosseByPlayer(this.controller);
```

Comment:
```suggestion
        const posse = this.game.shootout.getPosseByPlayer(this.controller);
```
Variables should be lowercase.